extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php

```diff
@@ -76,6 +97,40 @@ private function getImageAsTitleObject( $imageName ) {
 		return $title;
 	}
 
+	private function getGalleryItems( $imageData ) {
+		preg_match( '/.(UNIQ.*QINU)./U', $imageData, $galleryMarkers );
+		$galleryImages = array();
+
+		foreach ( $galleryMarkers as $marker ) {
+			$galleryWikitext = PortableInfoboxDataBag::getInstance()->getGallery( $marker );
+			$galleryLines = preg_split('/\r\n|\n|\r/', $galleryWikitext);
+
+			foreach ( $galleryLines as $galleryImage) {
```

Comment:
@idradm @inez My proposition is to reuse the existing gallery parser which will save us lot of regexing and reduce costs of future maintenance in case of any HTML changes in any of multiple gallery types we support. Fast implementation here: https://github.com/Wikia/app/commit/63effc2c54019bf006c3310f93e13d8552ff5700